git-cliff-core/src/template.rs

```diff
@@ -143,7 +143,7 @@ impl Template {
 	}
 
 	/// Renders the template.
-	pub fn render<C: Serialize, T: Serialize, S: Into<String> + Copy>(
+	pub fn render<C: Serialize, T: Serialize, S: Into<String> + Clone>(
```

Comment:
this change is needed because String doesn't implement Copy

Reply:
Any idea if this will have some side effects?

Reply:
This function is exported to the users of the library so it's a breaking change. People who are calling `render` will need to call `.clone()` I think 🤔

Reply:
I think it's fine since `git-cliff-core` does not have many reverse dependencies.